src/app/containers/schedule/daily-schedule/daily-schedule.component.ts

```diff
@@ -33,18 +33,18 @@ export class DailyScheduleComponent implements OnInit, OnDestroy {
     private schedule: ScheduleService) {}
 
   ngOnInit() {
-    this.buildDailySchedule();
-
     this.subjects = this.schedule.getSubjects();
     this.teachers = this.schedule.getTeachers();
 
+    this.buildDailySchedule();
+
     for (let i = 0; i < this.lessonsMaxPerDay; i++) {
       this.secondGroupVisible.push(false);
```

Comment:
weird a bit, but might be replaced with this.secondGroupVisible - `new Array(this.lessonsMaxPerDay).fill(false)`

Reply:
ok. used it. saved it ti next commit.